type-definitions/ts-tests/updateTypes.ts

```diff
@@ -0,0 +1,17 @@
+import { NoUpdate, Update, SideEffects, UpdateWithSideEffects } from 'react-recomponent';
```

Comment:
Just a side-node: Do you know if prettier supports TypeScript as well? If so we should probably run it for those files as well just for completeness 🙂

Reply:
It does.
You can also add `husky` and `lint-staged` to run prettier on the commit hook